addons/mail/controllers/thread.py

```diff
@@ -8,6 +8,22 @@
 
 
 class ThreadController(http.Controller):
+    @http.route("/mail/thread/access_rights", methods=["POST"], type="json", auth="user")
+    def mail_thread_access_rights(self, thread_data):
```

Comment:
Please give a docstring, especially to explain 'thread_data' and its expected format